Gemfile

```diff
@@ -64,6 +64,7 @@ group :development, :test do
 end
 
 group :development do
+  gem 'therubyracer'
```

Comment:
Shouldn't we install it directly in our Docker image? I mean, install the NodeJS there directly instead of adding this here?

Reply:
Well, that way we make the application more independent of the platform it's running on. However I don't mind, what ever makes more cense for a ruby application

Reply:
I am not against the Docker Anderson. I think we could have the real NodeJS instead of this therubyracer.

Reply:
You can install the NodeJS directly in your docker image.